packages/webvh/src/dids/WebVhDidRegistrar.ts

```diff
@@ -0,0 +1,207 @@
+import type {
+  AgentContext,
+  DidCreateOptions,
+  DidCreateResult,
+  DidDeactivateOptions,
+  DidDeactivateResult,
+  DidRegistrar,
+  DidUpdateOptions,
+  DidUpdateResult,
+  VerificationMethod,
+} from '@credo-ts/core'
+
+import { DidRepository, KeyType, DidDocument, DidRecord, DidDocumentRole, JsonTransformer } from '@credo-ts/core'
+import { type DIDLog, createDID, multibaseEncode, MultibaseEncoding, updateDID } from 'didwebvh-ts'
+
+import { WebvhDidCrypto } from './WebvhDidCrypto'
+import { WebvhDidCryptoSigner } from './WebvhDidCryptoSigner'
+
+interface WebVhDidCreateOptions extends DidCreateOptions {
+  domain: string
+  paths?: string[]
```

Comment:
I know using an array is handy because didwebvh-ts uses it like that, but don't you think it would be better for us to consider the path as a single string that contains as many as subpaths as needed (e.g. path: 'users/alice' ? This way there will be a single path tag to add to the record as well